curriculum/challenges/english/01-responsive-web-design/applied-accessibility/give-links-meaning-by-using-descriptive-link-text.md

```diff
@@ -15,11 +15,11 @@ Screen readers do this by reading the link text, or what's between the anchor (`
 
 # --instructions--
 
-The link text that Camper Cat is using is not very descriptive without the surrounding context. Move the anchor (`a`) tags so they wrap around the text "information about batteries" instead of "Click here".
+The link text that Camper Cat is using is not very descriptive without the surrounding context. Move the anchor (`a`) tags so they wrap around the text "`information about batteries`" instead of "`Click here`".
```

Comment:
```suggestion
The link text that Camper Cat is using is not very descriptive without the surrounding context. Move the anchor (`a`) tags so they wrap around the text `information about batteries` instead of `Click here`.
```
As Randy said, it makes sense to have either `"quoted string in code"`, "quoted string in text" or just `code`, but "`quoted code`" both looks weird and is a bit ambiguous.